### PR TITLE
adds liquid laughter

### PR DESCRIPTION
**Pull request by 1fbff5f83b23d39d38b1dfcb4cac8d9b**
Adds a chemical that causes affected people to laugh. Now somebody will actually laugh at your jokes!
(Sorry for using upload instead of git, I'm a bit short on time)

:cl:
rscadd: Mix two of the best things in life (banana juice and sugar) to create pure happiness!
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
https://github.com/tgstation/tgstation/pull/19405

**Comment by Super3222** [comment]
Upload isn't going to work the way you want it too... because now you've re-added an entire file w/ those canned laughter changes. What you'd need to do is to copy and paste the changes you want into the file.

**Comment by McDonald072** [comment]
I don't see that in our code, though? Mind pointing me to where it is?

**Comment by McDonald072** [comment]
@Super3222 I'll fix that tomorrow

**Comment by X-TheDark** [comment]
other_reagents.dm is located [here](https://github.com/yogstation13/yogstation/blob/master/code/modules/reagents/chemistry/reagents/other_reagents.dm)

others.dm is located [here](https://github.com/yogstation13/yogstation/blob/master/code/modules/reagents/chemistry/recipes/others.dm)

**Comment by X-TheDark** [comment]
And you do need to delete those two files you uploaded...seeing as you uploaded them into the [main folder of the code](https://github.com/1fbff5f83b23d39d38b1dfcb4cac8d9b/yogstation/tree/eb04c2f3c63835e96ca1ef5d8b7f59162226d056)

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1247,3 +1247,29 @@ datum/reagent/romerol
...
+	..()
+
+/datum/reagent/laughter/on_mob_delete(mob/living/M)
+	M << "<span class='notice'>Everything is terrible again...</span>"
+
+/datum/reagent/laughter/overdose_start(mob/living/M)
+	M << "<span class='userdanger'>You start laughing hysterically!</span>"
+
+/datum/reagent/laughter/overdose_process(mob/living/M)
+	M.emote(pick(list("laugh","giggle")))
```
Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1275]: these are the same as the regular on_life emotes though

Comment (McDonald072) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1275]: When overdosed you laugh every tick, normally you laugh about once in 4. 
But I could add more emotes or effects, why not

**Pull request merged**